hyp3_gamma/__main__.py

```diff
@@ -183,6 +184,13 @@ def water_map():
     parser.add_argument('--hand-fraction', type=float, default=0.8)
     parser.add_argument('--membership-threshold', type=float, default=0.45)
     parser.add_argument('granule')
+
```

Comment:
Is it redundant to include default values since they already exist in the job spec? Maybe @asjohnston-asf can weigh in.

Reply:
I think default values here are preferred, since it's nice to have reasonable examples/default when running the python code or docker container outside of HyP3. I agree it's unfortunate that the defaults have to be written down in two different places, though. We went ahead and lived with the duplication for RTC_GAMMA, for example:
https://github.com/ASFHyP3/hyp3-gamma/blob/develop/hyp3_gamma/__main__.py#L57
https://github.com/ASFHyP3/hyp3/blob/develop/job_spec/RTC_GAMMA.yml#L30

Reply:
Gotcha, that's what I figured.